dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -132,6 +133,7 @@ private static ArtifactResults generateReport(Configuration configuration, Path
       Map<Artifact, Artifact> upperBoundFailures =
           findUpperBoundsFailures(completeDependencies, transitiveDependencies);
 
+      String dependencyTree = DependencyTreeFormatter.formatDependencyPaths(completeDependencies.list());
```

Comment:
Not in this PR, but I suspect we're going to want this to return a more complex data structure in the future that can be more neatly formatted in HTML

Reply:
I agree. I think ul/li tags with collapse/expand would be nice. e.g., `formatDependencyPathsHtml`